src/layouts/Login/LoginPage.tsx

```diff
@@ -233,51 +240,43 @@ export const LoginPage = (): JSX.Element => (
     <RestrictedGovtMasthead />
     <Grid {...LOGIN_GRID_LAYOUT}>
       <GridItem area="image" bgColor="primary.500">
-        <Flex
-          h="calc(100vh - 5rem - 2rem)"
+        <VStack
+          h="100%"
           w="100%"
-          alignItems="end"
-          justifyContent="start"
-          position="relative"
+          alignItems="center"
+          justifyContent="center"
+          px="10%"
         >
           <Text
             fontSize="2.25rem"
             color="base.content.inverse"
             textStyle="h2"
-            position="absolute"
-            maxW="14.25rem"
-            top="5vh"
-            right="5.5rem"
+            maxW="90%"
+            pb="3.5rem"
           >
             Rapidly build & launch informational sites
           </Text>
-          <LoginImage maxH="100%" maxW="100%" />
-        </Flex>
+          <LoginImage maxH="100%" />
+        </VStack>
       </GridItem>
       <GridItem area="content" bgColor="white">
-        <Flex
-          h="100%"
-          maxW="54rem"
-          alignItems="start"
-          justifyContent="center"
-          pt="5vh"
-        >
+        <Flex h="100%" alignItems="start" justifyContent="center" pt="5vh">
           <LoginContent />
         </Flex>
       </GridItem>
       {/* Custom colour to match stairs */}
-      <GridItem area="credits" bgColor="#E9F0FB" borderColor="#E9F0FB">
+      <GridItem area="credits" bgColor="primary.500" borderColor="primary.500">
```

Comment:
uhm... does we have colour pallets for this ah? if have please use them instead

Reply:
Replaced the whites with pallettes, primary.500 is our palette here i think! https://github.com/isomerpages/isomercms-frontend/pull/1619/commits/34effe40ac540c5a6d506143c4e139d3830644a8